bod-endava/e-toast-react
# Get rid of NPM as package manager

## Ticket description: 
As a development team, we want to work with only one node package manager, either `npm` or `yarn` so we can prevent conflicts between two different lockfiles in the same repo. 

### **Acceptance Criteria:** 
- The repository only uses an specific version of `yarn` or `npm`. 
- Any other lockfile different from the chosen tool will be deleted. 
- The main documentation is updated so any new developer will be noticed about the version of `npm` or `yarn` they must need to install in their local machine.

## **Testing**:
First of all, make sure you are using `yarn` version 1.22.x (see the README)
 - If you have the project already cloned in your local machine, run `rm -rf node_modules` 
 - Run `yarn install && yarn build`
 - Run `yarn test` and make sure everything is OK. 
 - Run `yarn storybook` and make sure the storybook app is running correctly. 


Cheers! 👍 🥂
